src/Toolchain/ToolchainProvider.ts

```diff
@@ -311,40 +311,23 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
       vscode.window.showInformationMessage("Onecc has run successfully.");
     };
 
-    /* istanbul ignore next */
-    const notifyGuideline = () => {
-      this.error(
-        "Default toolchain is not set. Please install toolchain and set the default toolchain.",
-        "OK",
-        "See Instructions"
-      ).then((value) => {
-        if (value === "See Instructions") {
-          /* istanbul ignore next */
-          DefaultToolchain.getInstance().openDocument();
-        }
-      });
-    };
-
     /* istanbul ignore next */
     const notifyError = () => {
       this.error("Running onecc has failed.");
     };
 
-    const activeToolchainEnv = DefaultToolchain.getInstance().getToolchainEnv();
-    const activeToolchain = DefaultToolchain.getInstance().getToolchain();
-
-    if (!activeToolchainEnv || !activeToolchain) {
-      notifyGuideline();
+    const [toolchainEnv, toolchain] = this.checkAvailableToolchain();
+    if (toolchainEnv === undefined || toolchain === undefined) {
```

Comment:
Q) any particular reason to change variable name without `active`?
(I assume this `toolchain` is not related with active or not...)

Reply:
There is no special reason to change the name. I will put the name back. :)